Swiper 6.4.0 typings errors

This is a:
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 6.4.0

### What you did
I updated to 6.4.0

### Expected Behavior
No typescript compilation error.

### Actual Behavior
```
node_modules/swiper/types/swiper-class.d.ts:372:17 - error TS7006: Parameter 'breakpoints' implicitly has an 'any' type.

372   getBreakpoint(breakpoints): string;
                    ~~~~~~~~~~~
Found 1 error.
```

--- a/src/types/swiper-class.d.ts
+++ b/src/types/swiper-class.d.ts
@@ -369,7 +369,7 @@ interface Swiper extends SwiperClass<SwiperEvents> {
 
   isHorizontal(): boolean;
 
-  getBreakpoint(breakpoints): string;
+  getBreakpoint(breakpoints: SwiperOptions['breakpoints']): string;
 
   setBreakpoint(): void;
 

